app/models/manual.rb

```diff
@@ -0,0 +1,163 @@
+      Section.find(content_id: content_id)
+    }
+  end
+
+  def organisations=(organisation_content_ids)
+    @organisation_content_ids = organisation_content_ids
+  end
+
+  OrganisationStruct = Struct.new(:content_id, :base_path, :title)
+
+  def organisations
+    @organisations ||= @organisation_content_ids.map { |content_id|
+      payload = publishing_api.get_content(content_id).to_hash
+      OrganisationStruct.new(payload["content_id"], payload["base_path"], payload["title"])
+    }
+  end
+
+  def self.all
+    # Fetch individual payloads and links for each `manual`
+    payloads = self.content_ids.map { |payload|
+      publishing_api.get_content(payload.content_id).to_hash.deep_merge!(
+        publishing_api.get_links(payload.content_id).to_hash
+      )
+    }
+
+    # Deserialize the payloads into real Objects and return them
+    payloads.map { |payload| self.from_publishing_api(payload) }
+  end
+
+  def self.where(organisation_content_id:)
+    # Fetch individual links for each `manual`
+    payloads = self.content_ids.map { |payload|
+      publishing_api.get_links(payload.content_id).to_ostruct
+    }
+
+    # Select ones which have the same `content_id` as the `organisation_content_id` arguement
+    payloads.select! { |payload| payload.links.organisations.present? }
+    payloads.select! { |payload| payload.links.organisations.include?(organisation_content_id) }
+
+    # Fetch the content_id
+    payloads = payloads.map { |payload|
+      content = publishing_api.get_content(payload.content_id).to_hash
+      content.deep_merge!(payload.links)
+    }
+
+    # Deserialize the payloads into real Objects and return them
+    payloads.map { |payload| self.from_publishing_api(payload) }
+  end
+
+  def self.find(content_id:, organisation_content_id: nil)
+    if organisation_content_id
+      links = publishing_api.get_links(content_id).o_struct
+
+      if links.organisations.include?(organisation_content_id)
+        content = publishing_api.get_links(content_id).to_hash
+        payload = content.deep_merge(links.to_hash)
+      else
+        return nil
+      end
+    else
+      content = publishing_api.get_content(content_id).to_hash
+      links = publishing_api.get_links(content_id).to_hash
+      payload = content.deep_merge(links)
+    end
+
+    self.from_publishing_api(payload)
+  end
+
+  def self.from_publishing_api(payload)
+    manual = self.new(
+      {
+        content_id: payload["content_id"],
+        title: payload["title"],
+        summary: payload["description"],
+        body: payload["details"]["body"],
+        publication_state: payload["publication_state"],
+        public_updated_at: payload["public_updated_at"],
+      }
+    )
+
+    manual.base_path = payload["base_path"]
+    manual.update_type = payload["update_type"]
+
+    if payload["links"]
+      manual.organisations = payload["links"].fetch("organisations", [])
+      manual.sections = payload["links"].fetch("sections", [])
+    end
```

Comment:
I'd say all of it.  Even though it's the same repo there's no git history to trace back from this new version to the old version for anyone from the future to trace back and unpick even though it's the intended behaviour.